### DISCUSSION TITLE
İbax kaç dolar $ , tahmininiz nedir?

### DISCUSSION THREAD

**Discussion by hsyntpl** [IBAX-io/go-ibax#1514] [Soru Cevap - Türkçe] [answered]
İbax kaç dolar $ , tahmininiz nedir?

**Comment by tplayse** [comment, marked as answer]
Uzun vadede 50 -100 $

**Reply by ghost** [reply]
2

**Comment by Yusufknk** [comment]
Bunu ilerleyen zamanlarda yaşanılan gelişmelerle birlikte tecrübe edeceğiz.

**Comment by jedac** [comment]
Çok övülüyo her yerde bahsediliyor ama bakalım neler yapacak uzun vade beklemekten yanayım

**Comment by aliumut52** [comment]
Maxsumum 1 dolar

**Comment by Agahefendi1** [comment]
>1 usd

**Comment by Ozlozlm** [comment]
3 dolar

**Comment by Abdullahoda** [comment]
0.13 dolar ile piyasaya girecek diye biliyorum

**Comment by Sbzkt** [comment]
> İbax kaç dolar $ , tahmininiz nedir?

İzlediğim you tube kanalında en az 1000 dolar kazanılır demişlerdi ama tam olarak kaç puana verdiklerini kimse bilmiyor sanırım
